packages/react-devtools-shared/src/devtools/views/Profiler/utils.js

```diff
@@ -99,21 +99,21 @@ export function prepareProfilingDataFrontendFromBackendAndStore(
     );
   });
 
-  return {dataForRoots};
+  return {dataForRoots, imported: false};
 }
 
 // Converts a Profiling data export into the format required by the Store.
-export function prepareProfilingDataFrontendFromExport(
-  profilingDataExport: ProfilingDataExport,
+export function prepareProfilingDataFrontendFromImport(
+  profilingDataImport: ProfilingDataExport,
```

Comment:
This isn't really correct. That data you're receiving as a parameter is the previously *exported data*. You're *returning* "imported" data I guess, but...

Reply:
But? When data is readed from previously exported data it becomes imported data. It may be subjective because the variable name can be explained as a logical entity and not a semantical so I can remove this if it is not incorrect.

Reply:
That's *after* it's been processed though. The two formats are different because of serialization constraints. (That's why there are two separate flow types for them.)